ui/proposal_details_page.go

```diff
@@ -415,9 +515,154 @@ func (pg *proposalDetails) Layout(gtx C) D {
 			},
 		}
 		return common.SubPageLayout(gtx, page)
+	})
+
+	if pg.showVoteOptions {
+		return pg.voteOptionsLayout(gtx)
+	}
```

Comment:
This should be handled using the modal interface. The layout and handlers for the modal should not be in this file. Create a modal for this.

Reply:
Checkout this modal https://github.com/planetdecred/godcr/blob/master/ui/info_modal.go